frontend/src/components/dashboard/Analysis.tsx

```diff
@@ -0,0 +1,432 @@
+            font: {
+              size: 25,
+            },
+            click: onClickLabel,
+            enter: onEnterLabel,
+            leave: onLeaveLabel,
+          };
+
+          return split.type === SplitType.horizontal
+            ? [
+                {
+                  type: "line",
+                  xMin: (ctx) => ctx.chart.scales.x.min,
+                  xMax: (ctx) => ctx.chart.scales.x.max,
+                  yMin: split.y,
+                  yMax: split.y,
+                  backgroundColor: "rgba(255, 99, 132, 0.25)",
+                },
+                {
+                  ...labelProps,
+                  xValue: (ctx) => ctx.chart.scales.x.min,
+                  yValue: split.y,
+                  position: {
+                    x: "end" as const,
+                    y: "center" as const,
+                  },
+                },
+              ]
+            : [
+                {
+                  type: "line",
+                  xMin: split.x,
+                  xMax: split.x,
+                  yMin: (ctx) => ctx.chart.scales.y.min,
+                  yMax: (ctx) => ctx.chart.scales.y.max,
+                },
+                {
+                  ...labelProps,
+                  xValue: split.x,
+                  yValue: (ctx) => ctx.chart.scales.y.min,
+                  position: {
+                    x: "center" as const,
+                    y: "start" as const,
+                  },
+                },
+              ];
+        }),
+        ...groups.flatMap<AnnotationOptions>((group) => [
+          {
+            type: "label" as const,
+            content: group.label,
+            color: groupLabelColor,
+            xValue: (ctx) => {
+              const min = Math.max(group.minX, ctx.chart.scales.x.min);
+              const max = Math.min(group.maxX, ctx.chart.scales.x.max);
+
+              return min + (max - min) / 2;
+            },
+            yValue: (ctx) => {
+              const min = Math.max(group.minY, ctx.chart.scales.y.min);
+              const max = Math.min(group.maxY, ctx.chart.scales.y.max);
+
+              return min + (max - min) / 2;
+            },
+            padding: 0,
+            font: {
+              size: 25,
+            },
+          },
+        ]),
+      ],
+    }),
+    [splits, setSplits, groups],
+  );
+
+  const chartOptions = useMemo<ChartConfiguration<"bubble">["options"]>(
+    () => ({
+      plugins: {
+        legend: {
+          display: true,
+          position: "right",
+          align: "center",
+        },
+
+        select: {
+          onAddSplit: (split) => {
+            setSplits((splits) => [...splits, split]);
+          },
+        },
+
+        tooltip: {
+          enabled: false,
+          position: "nearest",
+          external: onTooltip,
+        },
+
+        annotation: annotations,
+      },
+      scales: {
+        x: xAxisConfig,
+        y: yAxisConfig,
+      },
+      /* seems buggy */
+      animation: false,
```

Comment:
should this comment be `disable animation because its behavior seems buggy (Tyratox, 2024-12-04)` ?